### PR TITLE
Logs Updated

### PR DESCRIPTION
**Pull request by AliRKat**
formatting and updating the logs. in few places it was using deprecated configuration field, changed those into getter methods

### CODE REVIEW

**Comment by AliRKat** [comment]
[[Unity] Add brackets to logs that are missing it](https://trello.com/c/ir3SdwDf/1893-unity-add-brackets-to-logs-that-are-missing-it)

File: Assets/Countly/Plugins/CountlySDK/Services/DeviceIdCountlyService.cs [hunk 1]
```diff
@@ -65,9 +65,9 @@ internal void InitDeviceId(string deviceId = null)
                     DeviceIdType = (DeviceIdType)storedDIDType;
                 } else {
                     if (storedDIDType == DEVICE_TYPE_FALLBACK_VALUE) {
-                        Log.Error("[DeviceIdCountlyService] InitDeviceId: SDK doesn't have device ID type stored. There should have been one.");
+                        Log.Error("[DeviceIdCountlyService] InitDeviceId, SDK doesn't have device ID type stored. There should have been one.");
                     } else {
-                        Log.Error("[DeviceIdCountlyService] InitDeviceId: The stored device id type wasn't valid ['" + storedDeviceId + "']. SDK will assign a new type");
+                        Log.Error($"[DeviceIdCountlyService] InitDeviceId, The stored device id type wasn't valid [storedDeviceId: {storedDeviceId}]. SDK will assign a new type");
```
Comment (arifBurakDemiray) [inline Assets/Countly/Plugins/CountlySDK/Services/DeviceIdCountlyService.cs]: storeDeviceId: [value]
I will add a doc entry for the parameters

File: Assets/Countly/Plugins/CountlySDK/Services/RemoteConfigCountlyService.cs [hunk 2]
```diff
@@ -71,8 +69,7 @@ private Dictionary<string, object> FetchConfigFromDB()
                 config = Converter.ConvertJsonToDictionary(allConfigs[0].Json, Log);
             }
 
-            Log.Debug("[RemoteConfigCountlyService] FetchConfigFromDB : Configs = " + config);
-
+            Log.Debug($"[RemoteConfigCountlyService] FetchConfigFromDB, Configs: [{config}]");
```
Comment (arifBurakDemiray) [inline Assets/Countly/Plugins/CountlySDK/Services/RemoteConfigCountlyService.cs:72]: plural

File: Assets/Countly/Plugins/CountlySDK/Services/SessionCountlyService.cs [hunk 3]
```diff
@@ -41,7 +41,7 @@ internal SessionCountlyService(CountlyConfiguration configuration, CountlyLogHel
             _monoBehaviour = monoBehaviour;
 
             if (_configuration.IsAutomaticSessionTrackingDisabled) {
-                Log.Verbose("[Countly][CountlyConfiguration] Automatic session tracking disabled!");
+                Log.Verbose("[SessionCountlyService] Automatic session tracking disabled!");
```
Comment (arifBurakDemiray) [inline Assets/Countly/Plugins/CountlySDK/Services/SessionCountlyService.cs:44]: mb we could add constructor for those

Comment (arifBurakDemiray) [reply Assets/Countly/Plugins/CountlySDK/Services/SessionCountlyService.cs:44]: @turtledreams what do you think?

File: Assets/Countly/Plugins/CountlySDK/Services/RemoteConfigCountlyService.cs (hunk 2 above)
Comment (arifBurakDemiray) [reply Assets/Countly/Plugins/CountlySDK/Services/RemoteConfigCountlyService.cs:72]: I will fix this it is short

Comment (arifBurakDemiray) [reply Assets/Countly/Plugins/CountlySDK/Services/RemoteConfigCountlyService.cs:72]: or not it is param name ok

File: Assets/Countly/Plugins/CountlySDK/Services/DeviceIdCountlyService.cs (hunk 1 above)
Comment (arifBurakDemiray) [reply Assets/Countly/Plugins/CountlySDK/Services/DeviceIdCountlyService.cs]: I did this, it was a short thing

**Review by arifBurakDemiray — APPROVED** [review]
Thank you for your work, I liked the branch name

File: Assets/Countly/Plugins/CountlySDK/Services/SessionCountlyService.cs (hunk 3 above)
Comment (turtledreams) [reply Assets/Countly/Plugins/CountlySDK/Services/SessionCountlyService.cs:44]: too much work, could be sth we do if we really have nothing to do

Comment (turtledreams) [reply Assets/Countly/Plugins/CountlySDK/Services/SessionCountlyService.cs:44]: lets not give so much detail

**Review by turtledreams — APPROVED** [review]

**Pull request merged**